Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -153,7 +153,8 @@ private fun MapViewContainer(
     latitude: String,
     longitude: String
 ) {
-    var zoom by rememberSaveable { mutableStateOf(InitialZoom) }
+    var zoom by rememberSaveable(map) { mutableStateOf(InitialZoom) }
+    var mapInitialized by rememberSaveable(map) { mutableStateOf(false) }
```

Comment:
This should just be `remember` right? You'll need to re-initialize after activity death

Reply:
Sure, done! Thanks